packages/amplify_datastore/lib/method_channel_datastore.dart

```diff
@@ -54,12 +55,23 @@ class AmplifyDataStoreMethodChannel extends AmplifyDataStore {
       case 'errorHandler':
         Map<String, dynamic> arguments =
             Map<String, dynamic>.from(call.arguments);
-        _errorHandler!(_deserializeExceptionFromMap(arguments));
+        errorHandler!(_deserializeExceptionFromMap(arguments));
         break;
 
       case 'conflictHandler':
-        break;
+        Map<String, dynamic> arguments =
+            Map<String, dynamic>.from(call.arguments);
+
+        String modelName = arguments["modelName"];
+        var modelType = modelProvider!.getModelTypeByModelName(modelName);
```

Comment:
Be consistent - either `Type name = ...` or `var name = ...`, but preferably use `final name = ...` for values which don't change:

```suggestion
        final modelName = arguments["modelName"] as String;
        final modelType = modelProvider!.getModelTypeByModelName(modelName);
```